src/app/features/debriefings/components/molecules/DebriefForm/DebriefForm.tsx

```diff
@@ -12,15 +13,24 @@ type Props = {
   isLoading?: boolean
 }
 
-const DebriefForm: React.FC<Props> = ({ caseId, initialValues, isLoading, onSubmit }) => (
-    <ScaffoldForm
-      showSpinner={ isLoading }
-      onSubmit={ onSubmit }
-      initialValues={ initialValues ?? { case: caseId } }
-    >
-      <ScaffoldFields {...createScaffoldProps(caseId, initialValues !== undefined) } />
-    </ScaffoldForm>
-  )
+const FormWithTooltip = styled.div`
+  form > div > div > div > div {
```

Comment:
This looks kinda hacky :)

Reply:
I agree, I was hoping you would know a better the way. It's inside AmsterdamScaffolds..